examples/rendercontrol_opengl/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Core Gui OpenGL Qml Quick)
```

Comment:
Since you are linking against the Qt Quick Controls 2 module, you should add it here as well.

Reply:
Or if this application doesn't use that module, you can remove it from the linked libraries instead.

Reply:
Fixed in 5a5ec64a736891f327445fe5381b8d9b7de2ebc9. I've set the QtControls Styles as well in the same way as your quick example to be consistent.